README.md

```diff
@@ -7,38 +9,23 @@
 
 ## Getting Started
 
-1. Create a new repository using this template.
-2. Clone the repository to your local machine.
-3. Install the dependencies preferably using `yarn` or `bun`.
-
-## Creating a new plugin
-
-- If your plugin is to be used as a slash command which should have faster response times as opposed to longer running GitHub action tasks, you should use the `worker` type.
-
-1. Ensure you understand and have setup the [kernel](https://github.com/ubiquity/ubiquibot-kernel).
-2. Update [compute.yml](./.github/workflows/compute.yml) with your plugin's name and update the `id`.
-3. Update [context.ts](./src/types/context.ts) with the events that your plugin will fire on.
-4. Update [plugin-inputs.ts](./src/types/plugin-inputs.ts) to match the `with:` settings in your org or repo level configuration.
+1. Install the dependencies preferably using `yarn` or `bun`.
+2. Copy `.dev.vars.example` to `.dev.vars` and fill the variables
+3. Generate Supabase types by running
+    ```shell
+    yarn prebuild
+    ```
+4. Run the project with `yarn wrangler`
 
-- Your plugin config should look similar to this:
+## Example configuration
 
 ```yml
-- plugin: <plugin-org/owner>/<plugin-repo-name>:compute.yml@development
-  name: plugin-name
-  id: plugin-name-command
-  description: "Plugin description" # small description of what the plugin does
-  command: "<regex for command>" # if you are creating a plugin with a slash command
-  example: "<example usage>" # how to invoke the slash command
-  with: # these are the example settings, the kernel passes these to the plugin.
-    disabledCommands: []
-    timers:
-      reviewDelayTolerance: 86000
-      taskStaleTimeoutDuration: 2580000
-    miscellaneous:
-      maxConcurrentTasks: 3
-    labels:
-      time: []
-      priority: []
+- plugin: ubiquibot/command-wallet
+  name: command-wallet
+  id: command-wallet
+  description: "Helps users setting their wallet to collect subsequent task rewards."
+  command: "/wallet"
+  example: "/wallet ubiquibot.eth"
```

Comment:
ENS is broken last I checked so use an address like the zero address.

Reply:
I used an arbitrary ENS to keep this functionality alive
https://github.com/ubiquibot/command-wallet/blob/fdc2cc70444431d5f5bcbb111dc55ce8cdde50a7/src/handlers/query-wallet.ts#L81
You'd prefer to have it not working at all instead? Or could we use our rpc package?

Reply:
If it works correctly then that's great. 

Using our rpc handler everywhere is ideal

Reply:
I had no luck when trying to use the rpc package so I want to do it in a separate PR. This endpoint was quite reliable during my tests which is why I kept it. You can even try it in my testing repo if you want.

Reply:
> I had no luck when trying to use the rpc package

What exactly went wrong? I've tried recently the latest published version (`1.1.0`) which seemed to work fine.

Reply:
@rndquu It was quite slow and eventually would yield no result. Might have installed the wrong version or did something wrong when I used it then.